L6_MaxProductOfThree.py

```diff
@@ -78,11 +78,15 @@ def solution(A):
     :param A: list[int]
     :return: [int]
     """
-    A.sort()
-    return max(
-        A[0] * A[1] * A[-1],
-        A[-3] * A[-2] * A[-1]
-    )
+    max=0
+    
+    a=sorted(a)
+    a.reverse()
```

Comment:
The `sorted` routine has a parameter 'reverse'.  So you can reduce this to a single line:

`a = sorted(a, reverse=True)`

Alternatively, you could not skip the reverse, and instead directly access the array from the other end, something like...

`max = (a[-1] * a[-2] * a[-3])`